graph/xml_parser.py

```diff
@@ -59,13 +59,12 @@ def get_used_rules(self):
                         id_rule=ruleResult.get('idref'),
                         id_def=check_content_ref.attrib.get('name'),
                         href=check_content_ref.attrib.get('href'),
-                        result=result.text))
+                        result=result.text
```

Comment:
Also add a trailing line, i.e. `result=result.text,`
That way, adding another member of the dictionary will be just about adding one more line.
Concerning those brackets, it is OK to close both of them at once to save vertical space.